### PR TITLE
Update inventory template for dbt2

### PR DESCRIPTION
**Pull request by mw2q**
Set a flag for dbt2 in the database nodes if it is set in the spec file.

### CODE REVIEW

File: edbdeploy/project.py [hunk 1]
```diff
@@ -755,7 +755,8 @@ def provision(self, env):
             pooler_type=self.terraform_vars['pooler_type'],
             replication_type=self.terraform_vars['replication_type'],
             ssh_priv_key=self.terraform_vars['ssh_priv_key'],
-            ssh_user=self.terraform_vars['ssh_user']
+            ssh_user=self.terraform_vars['ssh_user'],
+            dbt2=self.terraform_vars['dbt2'],
```
[the remark this answers is not in the document]
Comment (mw2q) [reply edbdeploy/project.py:759]: Ok, I'll test that change.  FWIW, it seemed to work ok, but I admit my Python is not the best.

[the remark this answers is not in the document]
Comment (mw2q) [reply edbdeploy/project.py]: Successfully tested on RA1 on AWS, and an RA1 variant with dbt2 enabled on Azure.

[the remark this answers is not in the document]
Comment (jt-edb) [reply edbdeploy/project.py]: Thank you!

**Review by jt-edb — APPROVED** [review]
LGTM

**Pull request merged**